doc/release-process.md

```diff
@@ -136,7 +136,7 @@ Commit your signature to guix.sigs:
 ```sh
 pushd ./guix.sigs
 git add "${VERSION}/${SIGNER}"/noncodesigned.SHA256SUMS{,.asc}
-git commit -m "Add ${VERSION} unsigned sigs for ${SIGNER}"
+git commit -m "Add ${SIGNER} Guix attestations for ${VERSION} non-codesigned"
```

Comment:
Don't think this needs to mention "Guix", it's the guix signatures repository so that is implicit.

Reply:
I am also thinking if the sentence was proper English before.

Shouldn't it say "Add attestations *by* $SIGNER for $VERSION non-codesigned"?

Reply:
Yes, that'd be even better I think.